component/suites/test.jsx

```diff
@@ -15,9 +15,7 @@ let Test = ({uuid, status, name, message, onToggle, collapsed}) => {
   let Content = null
   if (message) {
     Content = <div
-      className='card-content'
-      dangerouslySetInnerHTML={{__html: message}}
-    />
+      className='card-content'>{message}</div>
```

Comment:
this is what I was trying to explain which I think you will now be able to see

I generate the demo with

```
./bin/xunit-viewer --results=data  --ignore=invalid.xml,blank_file.xml --output=index.html
```

do that then search for the test `bacon 11`  then note the text `inner message`
that is using HTML as it's message and I want to be able to render it
this is useful when generating xunit reports for screenshot tests and you want to add a message with an image tag with the failing image

it should look like this, which I think your example will not

![image](https://user-images.githubusercontent.com/4129128/31496860-f04f178e-af54-11e7-95ef-0e791f6019e4.png)

Hence it might be good to add an option like `--escape` which might do something here like 

```jsx
{window.ESCAPE ?  <div className='card-content'><pre>{message}<pre></div> :<div className='card-content' dangerouslySetInnerHTML={{__html: message}} />}
```

you will need to change `cli/resolve-args.js` and `cli/resolve-options.js` in order to collect `escape` and make sure it defaults correctly just like `--watch`

then change `cli/template.html` and add something which sets this on the window as well as `cli/render.js` in order to pass in this option

escape should default to `false` so as to be backwards compatible, not necessarily because it makes sense (in hindsight I should have thought this through some more)

Reply:
just had another thought, you could wrap your message in a `<code>` block for a temporary fix e.g. `message='<code>SNAPSHOTSTUFF</code>'`

that said not sure how much control you have over jest/jasmine reporter output